src/concurrency/cross_thread_buffer.rs

```diff
@@ -0,0 +1,38 @@
+use rayon::iter::{IntoParallelRefMutIterator, ParallelBridge, ParallelIterator};
+use std::cell::RefCell;
+use thread_local::ThreadLocal;
+
+pub(crate) struct CrossThreadBuffer<T: Send> {
```

Comment:
What is this used for? (not sure if I missed its usage somewhere else in the PR)

Reply:
It's used for the buffer that stores dead data in the collector